util/pertubation.py

```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
+import librosa
+import numpy as np
+
+def speed_pertub(samples):
+    samples = samples.flatten().astype('float32')
+    speed_range = np.random.uniform(low=0.8, high=1.4)
+    samples = librosa.effects.time_stretch(samples, rate=speed_range)
+    return samples.reshape([-1, 1])
+
+def pertub(samples):
```

Comment:
Please give this a more descriptive name like `apply_random_time_stretch`.

Reply:
Seems like this file was leftover with your other changes and is no longer used.

Reply:
removed